### PR TITLE
Script updates - healthcheck, configs, cron tweaks

### PR DESCRIPTION
**Pull request by phidauex**
Hi, I've been tweaking and updating your very-helpful script over time, and thought I should clean it up and submit a PR of my changes. I'm a novice bash scripter, so please let me know if you see anything stupid.

This PR reorganizes the config variables, moves the default directory to its own variable so it is easier to see, adds Healthcheck.io notifications, adds a default terminal to prevent cron errors, and checks for an interactive terminal so that the `description` function can be left in place and the script will still work interactively and in cron without changes.

Update the docs to include the new features.

### CODE REVIEW

**Comment by DerDanilo** [comment]
Thanks for your PR. Very useful! So far it looks good to me.

Can you please fix the date? 

You may as well remove the "BETA" comment next to the script. It's productive for a while now and used by quiet a few people not reporting serious errors. I guess it's okay to declare it stable.

Thanks!

File: prox_config_backup.sh [hunk 1]
```diff
@@ -1,32 +1,55 @@
 #!/bin/bash
-# Version	      0.2.2 - BETA ! !
-# Date		      02.20.2020
+# Version	      0.2.3 - BETA !!
+# Date		      04.17.2020
```
Comment (DerDanilo) [inline prox_config_backup.sh]: Fix date.

File: prox_config_backup.sh [hunk 2]
```diff
@@ -1,32 +1,55 @@
 #!/bin/bash
-# Version	      0.2.2 - BETA ! !
-# Date		      02.20.2020
+# Version	      0.2.3 - BETA !!
```
Comment (DerDanilo) [inline prox_config_backup.sh]: Remove BETA tag

File: prox_config_backup.sh [hunk 3]
```diff
@@ -1,32 +1,55 @@
...
-# example: export BACK_DIR="/mnt/pve/media/backup"
-# or
-# example: BACK_DIR="." ./prox_config_backup.sh
-_bdir=${BACK_DIR:-/mnt/backups/proxmox}
+# Permanent backups directory
+# Default value can be overridden by setting environment variable before running prox_config_backup.sh
+#   example: export BACK_DIR="/mnt/pve/media/backup"
+#   or
+#   example: BACK_DIR="." ./prox_config_backup.sh
+DEFAULT_DIR="/mnt/pve/media/backup"
```
Comment (DerDanilo) [inline prox_config_backup.sh]: Please rename to `DEFAULT_BACK_DIR` (and all references of course)

**Review by DerDanilo — CHANGES_REQUESTED** [review]

File: prox_config_backup.sh (hunk 1 above)
Comment (phidauex) [reply prox_config_backup.sh]: Done!

File: prox_config_backup.sh (hunk 3 above)
Comment (phidauex) [reply prox_config_backup.sh]: Updated, good call to make the variable names more consistent. Updated in the README as well.

**Comment by phidauex** [comment]
Thanks, changes implemented! I updated the date to today - is that what you were intending?

**Comment by DerDanilo** [comment]
> Thanks, changes implemented! I updated the date to today - is that what you were intending?

Yes, you changed day and month but not year. GTM now.

**Pull request merged**